src/Sylius/Bundle/AddressingBundle/Doctrine/ORM/ZoneRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AddressingBundle\Doctrine\ORM;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Addressing\Repository\ZoneRepositoryInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ZoneRepository extends EntityRepository implements ZoneRepositoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function findOneByName($name)
+    {
+        return $this->createQueryBuilder('o')
+            ->addSelect('members')
+            ->leftJoin('o.members', 'members')
```

Comment:
:+1: Some of this mapping could be resolved by fetch eager, but it will be done in separated PR